Core/src/LongRunning/LROTraitV2.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Core\LongRunning;
+
+use Google\ApiCore\Serializer;
+use Google\Cloud\Core\Iterator\ItemIterator;
+use Google\Cloud\Core\Iterator\PageIterator;
+use Google\Cloud\Core\RequestHandler;
+# TODO: Point to the correct request class
```

Comment:
what is the correct request class?

Reply:
This is correct class, this comment added here to discuss further on the arrival of new [OperationsClient](https://github.com/googleapis/php-longrunning/blob/main/src/LongRunning/Client/OperationsClient.php)

I will remove this comment.